src/Compilation.zig

```diff
@@ -6924,6 +6924,11 @@ pub fn addCCArgs(
                 },
             }
 
+            switch (target.os.tag) {
+                .@"3ds" => try argv.append("-D__3DS__"),
+                else => {},
+            }
```

Comment:
Is this an established macro? E.g. used by homebrew toolchains?

Reply:
It's defined by default in every [makefile](https://github.com/devkitPro/3ds-examples/blob/master/templates/application/Makefile#L53) used to compile 3DS apps. I'd say it'd be an expected macro to have when targeting the os.